### ISSUE TITLE
Cannot find Fairemail in the app list

### ISSUE THREAD

**Issue by lichingyun** [OlauncherCF/OlauncherCF#233]
- [ X ] I have searched existing issues for this bug.

#### Bug description
I cannot find Fairemail in the app list in OCF, so I cannot add Fairemail to the homescreen 

#### Steps to reproduce

1. I have buy a new mobile phone and configure it
2. I have installed the latest version of OCF from F-Droid 
3. I have installed the latest Fairemail from Github
4. I have configure my homescreen but Fairemail is "invisible", using the original launcher of my phone is possible to see Fairemail with is  1.4.0

#### Environment information

- Olauncher CF version:
- Device name: Oneplus Nord 2 5g
- Android version: 12 (Oxygen OS 12.1)
- Other information: I don't have this problem using the latest OCF on my old Xiaomi Redmi Note 5 Whyred with Arrow Os 12.1 (Android 12)

#### Additional context (that might help us to solve your problem)

- Screenshots or other context

**Comment by Npa972** [comment]
Hi i use Fairemail with a xiaomi phone (android 11) and i don't have this issue. I can rename, hide... Fairemail without problem.
Fairemail from playstore.

**Comment by skdubg** [comment]
Fairemail is here too. no problems with 1.4.0

**Comment by lichingyun** [comment]
![Screenshot_2022-12-17-09-31-33-08_ef363686a81023f23b792a7426d86cce](https://user-images.githubusercontent.com/79099214/208233363-d29a7c23-67c7-469b-ad70-33b279b3764b.jpg)

Hi, I send here the screenshot, here is my apps list, but Fairemail is not showed

**Comment by jooooscha** [comment]
Maybe you have hidden it by mistake?

**Comment by lichingyun** [comment]
> Maybe you have hidden it by mistake?

no, I don't have hidden apps

**Comment by The-Repo-Club** [comment]
> > Maybe you have hidden it by mistake?
> 
> no, I don't have hidden apps I have try Mlauncher (it is a fork of Olauncher CF ) and Fairemail is showed in the app list

if it shows in mLauncher then it should show here as the code base is the same other then a few languages and i added up swipe (adding toggle for the auto open feature) but other then that its the same

**Comment by lichingyun** [comment]
> > > Maybe you have hidden it by mistake?
> > 
> > 
> > no, I don't have hidden apps I have try Mlauncher (it is a fork of Olauncher CF ) and Fairemail is showed in the app list
> 
> if it shows in mLauncher then it should show here as the code base is the same other then a few languages and i added up swipe (adding toggle for the auto open feature) but other then that its the same

I don't know what to say, I have installed both mLauncher and Olauncher CF but only mLauncher show Fairemail. I am not a programmer so I can't help with the code, sorry

**Comment by The-Repo-Club** [comment]
> > > > Maybe you have hidden it by mistake?
> > > 
> > > 
> > > no, I don't have hidden apps I have try Mlauncher (it is a fork of Olauncher CF ) and Fairemail is showed in the app list
> > 
> > 
> > if it shows in mLauncher then it should show here as the code base is the same other then a few languages and i added up swipe (adding toggle for the auto open feature) but other then that its the same
> 
> I don't know what to say, I have installed both mLauncher and Olauncher CF but only mLauncher show Fairemail. I am not a programmer so I can't help with the code, sorry

maybe uninstall and reinstall it and se if it shows up? maybe its just bugged out this one time?

**Comment by lichingyun** [comment]
> > > > > Maybe you have hidden it by mistake?
> > > > 
> > > > 
> > > > no, I don't have hidden apps I have try Mlauncher (it is a fork of Olauncher CF ) and Fairemail is showed in the app list
> > > 
> > > 
> > > if it shows in mLauncher then it should show here as the code base is the same other then a few languages and i added up swipe (adding toggle for the auto open feature) but other then that its the same
> > 
> > 
> > I don't know what to say, I have installed both mLauncher and Olauncher CF but only mLauncher show Fairemail. I am not a programmer so I can't help with the code, sorry
> 
> maybe uninstall and reinstall it and see if it shows up? maybe its just bugged out this one time?

I have try to install e reinstall but Fairemail always not showing

**Comment by The-Repo-Club** [comment]
i just installed Fairemail and OLCF and it works fine.
![sdasd](https://user-images.githubusercontent.com/9284733/208269716-2c49ae15-b3e8-4566-864e-8a5564129faf.png)

**Comment by jooooscha** [comment]
This is unlikely to be caused by fairmail itself, reinstalling that will not make a difference.

My first guess would be that you have hidden fairmail by mistake. But if it does not show up in the hidden apps list, I am would have to take a look at the device itself to debug this.

So, we should keep an eye open, if this happens repeatedly. But for now, reinstalling Olauncher will very likely fix this.

I am sorry for this inconvenience

**Comment by The-Repo-Club** [comment]
> This is unlikely to be caused by fairmail itself, reinstalling that will not make a difference.

ya i ment re install the launcher not fairmail

**Comment by jooooscha** [comment]
Could you report, if this fixed your issue, or if it still persists?

**Comment by lichingyun** [comment]
> Could you report, if this fixed your issue, or if it still persists?

Hi this issue still persists, I have try the latest OCF from F-Droid

**Comment by jooooscha** [comment]
Did you reinstall OlauncherCF?

**Comment by The-Repo-Club** [comment]
there shouldn't be any changes between mLauncher and OlauncherCF that would make this show up on mine and not here

**Comment by lichingyun** [comment]
> Did you reinstall OlauncherCF?

yes, I reinstall Olauncher CF

**Comment by jooooscha** [comment]
Mh, thats very weired. Maybe something with work profile? Otherwise, I have currently no idea what could have caused this

**Comment by The-Repo-Club** [comment]
> Mh, thats very weired. Maybe something with work profile? Otherwise, I have currently no idea what could have caused this

Tbh it's very wierd seeing as mLauncher doesn't have the same issue even though it's the same with just a few extra languages. 

I think I might do a meld on both projects and see if there is much different in any other places that I could of fixed or broke in some cases :)

**Comment by lichingyun** [comment]
@jooooscha @The-Repo-Club I have a question for you both... is an OT question so I beg your pardon 🙏 but... why don't you merge Olauncher CF and mLauncher? I see there are only few difference between this two projects

**Comment by jooooscha** [comment]
> why don't you merge Olauncher CF and mLauncher

That's exactly what i have been asking. And, until now, @The-Repo-Club never gave me an answer.

So to comment on the new statement: I do not remember having turned down any issue or feature request from @The-Repo-Club, so I am not sure why he forked OlauncherCF. I am especially confused, about the rebranding of OlauncherCF to mLauncher. As I see it, this only adds confusion, and does not reflect you stated intensions.

Anyway, as already said, I offered to move his modifications to into the OlauncherCF Organization, give it a fitting name that makes clear what the second version is supposed to be. However, @The-Repo-Club did not seem to be interrested.

To sum up, I think we should put both repos in one place and make it clear which version is which.

**Comment by The-Repo-Club** [comment]
> > why don't you merge Olauncher CF and mLauncher
> 
> That's exactly what i have been asking. And, until now, @The-Repo-Club never gave me an answer.
> 
> So to comment on the new statement: I do not remember having turned down any issue or feature request from @The-Repo-Club, so I am not sure why he forked OlauncherCF. I am especially confused, about the rebranding of OlauncherCF to mLauncher. As I see it, this only adds confusion, and does not reflect the stated intentions.
> 
> Anyway, as already said, I offered to move his modifications to into the OlauncherCF Organization, give it a fitting name that makes clear what the second version is supposed to be. However, @The-Repo-Club did not seem to be interested.
> 
> To sum up, I think we should put both repos in one place and make it clear which version is which.

well i am willing to move it over to  the OlauncherCF Organization as long as i still have control of it. also what would you call the 2nd version?

**Comment by jooooscha** [comment]
I opened #247 for further discussions
